run/pcap2john.py

```diff
@@ -1398,23 +1411,17 @@ def pcap_parser_htdigest(fname):
     f.close()
 
 
-def note():
-    sys.stderr.write("Note: This program does not have the functionality of wpapcap2john, SIPdump, eapmd5tojohn, and vncpcap2john.\n")
-
-
 ############################################################
 # original main, but now calls multiple 2john routines, all
 # cut from the original independent convert programs.
 ############################################################
 if __name__ == "__main__":
     if len(sys.argv) < 2:
-        sys.stderr.write("Usage: %s [.pcap files]\n\n" % sys.argv[0])
-        note()
-        sys.exit(-1)
+        sys.stderr.write("Usage: %s [.pcap files]\n" % sys.argv[0])
+        sys.exit(1)
 
     # advertise what is not handled
```

Comment:
This comment should probably have been moved along with the `note()` call.

Reply:
And perhaps the sleep should be moved too. Maybe both should be inside of `note()`.

Reply:
Ack for both. I am not planning to fix these issues at the moment.